sdk/eventhub/event-hubs/src/checkpointManager.ts

```diff
@@ -18,9 +18,9 @@ export interface Checkpoint {
    */
   consumerGroupName: string;
   /**
-   * @property The unique instance identifier
+   * @property The unique identifier of the event processor.
    */
-  instanceId: string;
+  ownerId: string;
```

Comment:
We need to update `event-hubs.api.md` file with this change

Reply:
Good catch!